src/commands/schema/status.mjs

```diff
@@ -15,33 +16,38 @@ async function doStatus(argv) {
   const logger = container.resolve("logger");
   const makeFaunaRequest = container.resolve("makeFaunaRequest");
 
-  let params = new URLSearchParams({ diff: "summary" });
   const secret = await getSecret();
+  const absoluteDirPath = path.resolve(argv.dir);
   const gatherFSL = container.resolve("gatherFSL");
   const fsl = reformatFSL(await gatherFSL(argv.dir));
 
+  const hasLocalSchema = fsl.entries().next().done === false;
```

Comment:
It might be simpler to check the results of `gatherFSL` instead. Does the same thing, though.

```js
  const gatherFSL = container.resolve("gatherFSL");
  const fslFiles = await gatherFSL(argv.dir)
  const fsl = reformatFSL(fslFiles);

  const hasLocalSchema = fslFiles.length > 0;
```

Reply:
nice